journals/apps/journals/journal_page_helper.py

```diff
@@ -20,3 +34,74 @@ def get_nested_children(self):
 
         structure["children"] = [child.specific.get_nested_children() for child in children]
         return structure
+
+    @property
+    def site(self):
+        """
+        Get the site associated with specific Journal page
+        """
+        journal = self.specific.get_journal()
+        if journal:
+            return journal.organization.site
+        else:
+            logger.error('No journal or site associated with page %s', self)
+            return None
+
+    def get_serializer(self, request):
+        """
+        Get the serializer for the given Page object using the same
+        mechanism used by the wagtail api.
+        Inspired by:
+        https://stackoverflow.com/questions/42539670/wagtail-serializing-page-model
+        """
+
+        # import here to avoid circular dependency
+        from journals.apps.api.v1.views import ManualPageSerializerViewSet
+        from journals.apps.api.v1.content.urls import wagtail_router
+
+        serializer_class = PagesAPIEndpoint._get_serializer_class(  # pylint: disable=protected-access
```

Comment:
I saw you posted the stackoverflow post above, but if you could write a little doc about what this is doing, that'd be helpful. In particular it'd be good to know what the `[('*', False, None)]` params stand for easier groking.